addons/website_slides/static/src/js/tours/slides_tour.js

```diff
@@ -100,11 +100,11 @@ wTourUtils.registerWebsitePreviewTour('slides_tour', {
     position: 'left',
     width: 170,
 }, {
-    trigger: ':iframe li.breadcrumb-item:nth-child(2)',
+    trigger: '.o_iframe[is-ready="true"]:iframe li.breadcrumb-item:nth-child(2)',
     content: markup(_t("Click on your <b>Course</b> to go back to the table of content.")),
     position: 'top',
 }, {
-    trigger: '.o_menu_systray_item a .o_switch',
```

Comment:
Is this due to the same change?

Reply:
It looks like this is actually coming from https://github.com/odoo/odoo/pull/151356 , it could be nice to include it as well in the PR message and commit message, as unrelated with other issue. 

Also, there are more than a single systray item... Could we make it more explicit and add 'form-switch' to indicate we only want to use the publish button?

Reply:
Actually, even better would be `trigger: ".o_menu_systray_item.o_website_publish_container a"`

Reply:
Hello @Odoonan 
How about this `.o_menu_systray_item a:contains(Unpublished)`

Reply:
Hello @thsh-odoo , it could work but let us go with `.o_menu_systray_item.o_website_publish_container a` in order to match other tours in event and shop ;) it works fine

Reply:
Selector is letting us know that we are clicking on unpublish toggle btn :smiley:

Reply:
Okay then :+1: